src/components/GetStarted.tsx

```diff
@@ -197,7 +205,7 @@ const GetStarted = ({
 						>
 							{deviceTypeSelector}
 							{intro}
-							<Heading.h3 bold my={3}>
+							<Heading.h3 bold mb={3} mt={4}>
```

Comment:
need to update this in `laptop and computer` tab as well.

Reply:
nice catch 👍